collector/mdadm_linux.go

```diff
@@ -138,12 +163,20 @@ func parseMdstat(mdStatusFilePath string) ([]mdStatus, error) {
 		}
 		currentMD = mainLine[0]               // name of md-device
 		isActive := (mainLine[2] == "active") // activity status of said md-device
+		personality = mainLine[3]             // Personality type of md-device.
```

Comment:
Ahh, yea, mainLine should always have atleast 4 parts anyway.  I'll update the test.